shared/js/media/media_frame.js

```diff
@@ -461,7 +461,8 @@ MediaFrame.prototype._switchToPreviewImage = function _switchToPreview() {
 
 MediaFrame.prototype.displayVideo = function displayVideo(videoblob, posterblob,
                                                           width, height,
-                                                          rotation)
+                                                          rotation, posterWidth,
+                                                          posterHeight)
```

Comment:
total nit: I'd put the rotation arg on the same line as width and height, so that posterWidth and posterHeight can be on the same line. Or if they don't fit on the same line, I'd put them on two separate lines by themselves.